cli/util.go

```diff
@@ -782,6 +782,11 @@ func renderCluster(cluster *api.ClusterInfo) string {
 	return f(compact)
 }
 
+// doReqAsyncWaitFullTimeoutInterval special value to be passed as `waitFor` parameter of doReqAsync, which results in
+// the latter function waiting for the Options.Timeout duration for replies. e.g. it turns off the "adaptive"
+// timeout behavior ("wait 300ms after the previous response then stop listening")
+const doReqAsyncWaitFullTimeoutInterval = -1
```

Comment:
imo this is harder to discover being here than just adding a comment to the `doReqAsync` function calling out this behaviour, this way it will be visible whenever someone uses the function but this constant would only be known to those going to look for it.

Reply:
Agreed, this is hard to discover and doesn't do much.

(I do like this constant for calling code, it's more readable than passing in a magic `-1`).

Updated to just document `doReqAsync`

Reply:
nice, could have kept the constant was mainly concerned with discoverability of the constant.  Looks good now will merge. Thank you!